### ISSUE TITLE
[Respects] Error with incorrect permissions

### ISSUE THREAD

**Issue by Injabie3** [SFUAnime/Ren#163]
```
[05/01/2019 00:49] ERROR red on_command_error 369: Exception in command 'f'
Traceback (most recent call last):
  File "lib/discord/ext/commands/core.py", line 50, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/home/ren/github/Ren/cogs/respects.py", line 55, in plusF
    if not await self.checkLastRespect(ctx):
  File "/home/ren/github/Ren/cogs/respects.py", line 164, in checkLastRespect
    exceedMessages = self.settings[sid][cid][KEY_MSG].id not in prevMsgs
AttributeError: 'NoneType' object has no attribute 'id'
```
Investigate further, but this is probably due to `renf` being invoked when the bot had no permissions to send message, making `self.settings[sid][cid][KEY_MSG]` be None, and causing this error to pop up.  We should handle this case properly.

**Comment by KaguneAstra** [comment]
Should be "No permissions to read messages" for this error. Should I fix the case where the bot has no permissions to send messages too?

**Comment by KaguneAstra** [comment]
I cannot replicate this error, I triggered the command while the bot had no permissions to send messages, then I proceeded to give it back permissions and triggered the command agian using another account. 
This error is different from the one stated above. I can fix this one but I have no idea for the one stated in this issue
![image](https://user-images.githubusercontent.com/33477372/61018693-94c08e00-a34c-11e9-88ed-6e8c59f11470.png)

**Comment by Injabie3** [comment]
As we discussed on discord, this was addressed in commit 35e97a3812fe31a8ac309124182e5a83b94bdc0d.  Forgot to update this issue.
